packages/cli/openapi-parser/src/schema/convertSchemas.ts

```diff
@@ -127,6 +128,11 @@ export function convertSchemaObject(
     const groupName = typeof mixedGroupName === "string" ? [mixedGroupName] : mixedGroupName;
     const generatedName = getGeneratedTypeName(breadcrumbs);
     const description = schema.description;
+    // TODO: style, map more availability types
```

Comment:
Are we planning to do this here, or in a follow-up?

Reply:
Thank you! Missed that one